src/main/java/net/openhft/chronicle/wire/WireMarshaller.java

```diff
@@ -1211,12 +1216,14 @@ protected void setValue(Object o, @NotNull ValueIn read, boolean overwrite) {
         }
 
         private void decodeBytes(@NotNull ValueIn read, Bytes<?> bytes) {
-            @NotNull StringBuilder sb0 = RSBP.acquireStringBuilder();
-            read.text(sb0);
-            String s = WireInternal.INTERNER.intern(sb0);
-            byte[] decode = Base64.getDecoder().decode(s);
-            bytes.clear();
-            bytes.write(decode);
+            try (ScopedResource<StringBuilder> stlSb = Wires.acquireStringBuilderScoped()) {
+                @NotNull StringBuilder sb0 = stlSb.get();
+                read.text(sb0);
+                String s = WireInternal.INTERNER.intern(sb0);
+                byte[] decode = Base64.getDecoder().decode(s);
+                bytes.clear();
+                bytes.write(decode);
```

Comment:
I wonder how often interning works with arbitrary base64 strings, do we not have a utility to decode base64 from a StringBuilder?